datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -1451,7 +1455,7 @@ type Role implements Entity {
     """
     Role properties to include Request Access Url
     """
-    properties: RoleProperties!
+    properties: RoleProperties
```

Comment:
Why is this no longer an optional field? What happens if someone wishes to create a role without properties?

Reply:
Hi @anshbansal we are facing issues in search and autocomplete in web-react search.graphql if we are making it as optional field. i.e why we update the same.